email_template_multi_report/models/email_template.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import base64
+
+from openerp import api, fields, models
+from openerp.tools.safe_eval import safe_eval
+
+
+class EmailTemplate(models.Model):
+
+    _inherit = 'email.template'
+
+    report_line_ids = fields.One2many(
+        'email.template.report.line', 'template_id', string='Other Reports')
+
+    @api.model
+    def generate_email_batch(self, template_id, res_ids, fields=None):
+        results = super(EmailTemplate, self).generate_email_batch(
+            template_id, res_ids, fields=fields)
+
```

Comment:
Please remove blank lines inside a method. It makes the reading harder as you need to scroll a lot, and it doesn't help too much for splitting logical code blocks (in this case, you have one line of code, one blank line). If I want to split a code block, I use instead a comment line explaining the purpose of that block.